crates/re_viewer/src/app.rs

```diff
@@ -507,13 +540,19 @@ impl eframe::App for App {
     fn update(&mut self, egui_ctx: &egui::Context, frame: &mut eframe::Frame) {
         let frame_start = Instant::now();
 
+        self.screenshotter.update(egui_ctx, frame);
+
         if self.startup_options.memory_limit.limit.is_none() {
             // we only warn about high memory usage if the user hasn't specified a limit
             self.ram_limit_warner.update();
         }
 
         #[cfg(not(target_arch = "wasm32"))]
-        {
+        if self.screenshotter.is_screenshotting() {
```

Comment:
Always resizing when taking a screenshot, even though command-palette feels unexpected.  I would expect this when using the command-line invocation but not when using the command palette.

Reply:
I've changed it to be a `--window-size` command line argument now!